torch/fx/experimental/proxy_tensor.py

```diff
@@ -173,11 +149,17 @@ def proxy_call(proxy_mode, func_overload, args, kwargs=None):
             "It's likely that this is caused by data-dependent control flow or similar."
         )
 
-    proxy_args, proxy_kwargs = pytree.tree_map_only(ProxyTensor, lambda e: e.proxy, (args, kwargs))
+    proxy_args, proxy_kwargs = pytree.tree_map_only(
+        SymInt,
+        fetch_symint_proxy(proxy_mode.tracer),
+        pytree.tree_map_only(ProxyTensor, lambda e: e.proxy, (args, kwargs))
+    )
     proxy_res = func_overload(*proxy_args, **proxy_kwargs)
 
     # Kind of a hacky way to test if an op is in-place or not
     if func.__name__[-1] == "_" and func.__name__[0] != "_":
+        # This makes DCE marginally less likely to DCE inplace operations.
```

Comment:
`inplace_view` tag ?

Reply:
Not views only, all inplace ops need this treatment.